src/connectors/breadcrumb/__tests__/connectBreadcrumb-test.js

```diff
@@ -1018,67 +789,40 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/breadcrumb/
     expect(helper.getHierarchicalFacetBreadcrumb('category')).toEqual([]);
   });
 
-  it('Provides a configuration if none exists', () => {
-    const makeWidget = connectBreadcrumb(() => {});
-    const widget = makeWidget({ attributes: ['category', 'sub_category'] });
+  describe('dispose', () => {
+    it('does not throw without the unmount function', () => {
+      const helper = jsHelper({}, '');
 
-    const partialConfiguration = widget.getConfiguration(
-      new SearchParameters()
-    );
+      const renderFn = () => {};
+      const makeWidget = connectBreadcrumb(renderFn);
+      const widget = makeWidget({ attributes: ['category'] });
 
-    expect(partialConfiguration).toEqual(
-      new SearchParameters({
-        hierarchicalFacets: [
-          {
-            attributes: ['category', 'sub_category'],
-            name: 'category',
-            rootPath: null,
-            separator: ' > ',
-          },
-        ],
-        hierarchicalFacetsRefinements: { category: [] },
-      })
-    );
-  });
-
-  it('Provides an additional configuration if the existing one is different', () => {
-    const makeWidget = connectBreadcrumb(() => {});
-    const widget = makeWidget({ attributes: ['category', 'sub_category'] });
+      expect(() =>
+        widget.dispose({ helper, state: helper.state })
+      ).not.toThrow();
+    });
 
-    const partialConfiguration = widget.getConfiguration(
-      new SearchParameters({
-        hierarchicalFacets: [
-          {
-            attributes: ['otherCategory', 'otherSub_category'],
-            name: 'otherCategory',
-            separator: ' > ',
-          },
-        ],
-      })
-    );
+    it('does not remove refinement', () => {
```

Comment:
not sure I get this part.
Removing the widget should remove the refinement, no?

Reply:
no, a breadcrumb's refinement is set by a configure or hierarchical facet, and should thus stay. This didn't change in this PR by the way, it was simply not tested